addons/pinniped/tanzu-auth-controller-manager/controllers/v1_cascade_controller_test.go

```diff
@@ -178,6 +178,10 @@ var _ = Describe("Controller", func() {
 		})
 
 		When("the secret is deleted", func() {
+			BeforeEach(func() {
+				deleteObject(ctx, secret)
+			})
```

Comment:
Wanted to call out that we don't actually need to delete the secret again here since it's deleted in an `aftereach` above, but it seemed good to be extra explicit.